docs/user-guide/import-certificates.md

```diff
@@ -0,0 +1,176 @@
+# Import and configure a certificate if you have one
+
+One option for using certificates in Zowe is to import and configure existing certificates. Zowe supports the following types of certificates for import:
+
+* File-based [PKCS12 certificates](#import-an-existing-pkcs12-certificate)
+* [JCERACFKS certificates](#import-an-existing-jceracfks-certificate)
+
+Review the procedure to import existing PKCS12 or JCERACFKS certificates as well as how to import a certificate stored in an MVS data set into a Zowe key ring.
+
+## Import an existing PKCS12 certificate
+
+To import a PKCS12 certificate, it is first necessary to [import a certificate authority](#manually-import-a-certificate-authority-into-a-web-browser). Then, you can configure the zowe.yaml according to [Scenario 2: Use a file-based (PKCS12) keystore and import a certificate generated by another CA](#certificate-configuration-scenarios.md/#scenario-2-use-a-pkcs12-keystore-and-import-a-certificate-generated-by-another-ca) described in the article Certificate configuration scenarios.
```

Comment:
This describes how to manually import a certificate to a browser. While this is also useful, it doesn't help at Zowe installation time. We need to talk about truststore and keyrings here.